internal/pkg/manifest/workload.go

```diff
@@ -290,13 +290,13 @@ func UnmarshalWorkload(in []byte) (interface{}, error) {
 
 	switch typeVal {
 	case LoadBalancedWebServiceType:
-		m := newDefaultLoadBalancedWebService()
+		m := &LoadBalancedWebService{}
```

Comment:
The reason why I change this is because currently when we are unmarshalling the manifest, we don't take the manifest as the source of truth. But I'm open for more input about it.